package.json

```diff
@@ -11,6 +11,7 @@
     "migrations": "knex migrate:latest",
     "recreate": "rm dev.sqlite3 && knex migrate:latest",
     "linter": "jshint ./src ./tests",
+    "fixtures": "node ./scripts/load_fixtures.js",
```

Comment:
Whenever I try to load the fixtures, I get an error that the activities table doesn't have a slug column. I don't recall the end of our discussion -- why did we want to move the slugs into their own table again?